### DISCUSSION TITLE
Bulk Renaming Exif Date Taken format

### DISCUSSION THREAD

**Discussion by Lockszmith-GH** [doublecmd/doublecmd#692] [General]
Anyone have any idea if the Exif Date Taken format can be manipulated in the bulk-rename dialog?
```
[=Plugin(<Exif>).Date taken{}]
```

Thanks in advance.

**Comment by alexx2000** [comment]
No, `Date taken` is stored as string and returned by plugin as is.

**Reply by Lockszmith-GH** [reply]
Thanks.
